.github/workflows/publish-kata-deploy-payload-amd64.yaml

```diff
@@ -21,46 +21,57 @@ on:
         required: false
         type: string
         default: ""
+      skip:
+        required: false
+        type: string
+        default: no
 
 jobs:
   kata-payload:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
+        if: ${{ inputs.skip != 'yes' }}
```

Comment:
Can we insert this condition at the job-level rather than step-level (for this and all other jobs)?

Reply:
We can but then the whole test gets skipped and we won't get the required tests PASS. Another solution is to double the jobs, which is something I also plan on investigating.